pkg/apis/minio.min.io/v2/constants.go

```diff
@@ -48,6 +48,9 @@ const MinIOCertPath = "/tmp/certs"
 // TmpPath /tmp path inside the container file system
 const TmpPath = "/tmp"
 
+// CfgPath is the location of the MinIO Configuration File
+const CfgPath = "/etc/minio/config.env"
```

Comment:
Let's avoid root path instead keep it at /tmp/config.env

Reply:
since I'm using volumes for this, I'll use different folders inside `/tmp`

Reply:
done